src/pages/Terms.tsx

```diff
@@ -68,8 +68,9 @@ const Terms: Component = () => {
                 Service in certain jurisdictions at any time.
             </p>
             <p>
-                4.4. You are at least 18 years old or of other legal age,
-                according to your relevant jurisdiction.
+                4.4. You are at least 18 years old or meet the higher legal age
```

Comment:
```suggestion
                4.4. You are at least 18 years old or meet the minimum legal age
```
IG this would be more accurate